### PR TITLE
1433 Add TorchVision transform wrapper

### LINKED ISSUE #1433: Add TorchVision transform to call TorchVision transforms
**Is your feature request related to a problem? Please describe.**
As `TorchVision` is very commonly used in DL projects, need to develop a wrapper transform to call TorchVision transforms based on both MONAI array and dict formats data.

Feature requested from internal team and users.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #1433 .

### Description
This PR added a wrapper transform for the TorchVision transforms.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh --codeformat --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/transforms/utility/array.py [hunk 1]
```diff
@@ -615,3 +616,32 @@ def __call__(
...
+            name: The transform name in TorchVision package.
+            args: parameters for the TorchVision transform.
+            kwargs: parameters for the TorchVision transform.
+
+        """
+        super().__init__()
+        transform, _ = optional_import("torchvision.transforms", name=name)
+        self.trans = transform(*args, **kwargs)
+
+    def __call__(self, img: torch.Tensor) -> torch.Tensor:
```
Comment (wyli) [inline monai/transforms/utility/array.py:641]: it seems torch vision doesn't have proper type hints for tensor/pil object, perhaps we could remove the typing for now?

Comment (Nic-Ma) [reply monai/transforms/utility/array.py]: Sure, updated in the latest commit.
Could you please help review it again?

Thanks.

**Review by wyli — APPROVED** [review]

**Pull request merged**